aquadoggo/src/node.rs

```diff
@@ -84,6 +85,11 @@ impl Node {
             panic!("Failed starting HTTP service");
         }
 
+        // Start libp2p service
+        if manager.add("libp2p", libp2p_service).await.is_err() {
+            panic!("Failed starting libp2p service");
```

Comment:
Similarily here we might want to give this service a different name here. People will look at it from the outside and wonder what this "libp2p" is doing